.github/actions/regression-tests/action.yml

```diff
@@ -56,6 +62,12 @@ runs:
         if [[ $code -ne 0 ]]; then
           exit 1
         fi
+      env:
+        # Add environment variables to enable the S3 snapshot test.
+        DRAGONFLY_S3_BUCKET: dragonflydb-regression-tests
```

Comment:
I would like to move it to secrets as well.

Reply:
sure updated